dist/css/bootstrap.css

```diff
@@ -3680,18 +3699,19 @@ fieldset[disabled] .checkbox label input[type=checkbox] + .input-replacement > s
 .checkbox > label,
 .radio > label {
   display: block;
-  padding-top: 12px;
+  padding-top: 11px;
   padding-right: 16px;
-  padding-bottom: 12px;
+  padding-bottom: 11px;
   font-size: 16px;
   line-height: 24px;
   color: #2f4366;
   cursor: pointer;
+  background-color: #fff;
   border: 1px solid #d3d5d8;
   border-radius: 3px;
-  -webkit-transition: border .1s ease-in-out;
-       -o-transition: border .1s ease-in-out;
-          transition: border .1s ease-in-out;
+  -webkit-transition: border .1s ease-in-out, background-color .1s ease-in-out;
+       -o-transition: border .1s ease-in-out, background-color .1s ease-in-out;
+          transition: border .1s ease-in-out, background-color .1s ease-in-out;
```

Comment:
Disregard last comments, did not check file name – is there a way to not show changes to compiled files?